crates/matrix-sdk-indexeddb/src/crypto_store/migrations.rs

```diff
@@ -48,28 +48,53 @@ pub async fn open_and_upgrade_db(
     // If we have yet to complete the migration to V7, migrate the schema to V6
     // (if necessary), and then migrate any remaining data.
     if old_version <= 6 {
+        info!(old_version, "IndexeddbCryptoStore upgrade schema & data -> v6 starting");
         let db = migrate_schema_up_to_v6(name).await?;
         migrate_data_for_v6(serializer, &db).await?;
         db.close();
+        info!(old_version, "IndexeddbCryptoStore upgrade schema & data -> v6 finished");
     }
 
     // Now we can safely complete the migration to V7 which will drop the old store.
+    if old_version < 7 {
+        migrate_schema_for_v7(name).await?;
+    }
+
+    // And finally migrate to v8, keeping the same schema but fixing the keys in
+    // inbound_group_sessions2
+    if old_version < 8 {
+        migrate_data_for_v8(name, serializer).await?;
+        migrate_schema_for_v8(name).await?;
+    }
+
+    // We know we've upgraded to v8 now, so we can open the DB at that version and return it
+    Ok(IdbDatabase::open_u32(name, 8)?.await?)
+}
+
+async fn migrate_schema_for_v8(name: &str) -> Result<(), DomException> {
+    info!("IndexeddbCryptoStore upgrade schema -> v8 starting");
+    IdbDatabase::open_u32(name, 8)?.await?;
+    // No actual schema change required for this migration
```

Comment:
it might be worth mentioning why we even bother with this migration (ie, as a flag that we have done the `migrate_data_for_v8` migration)

Reply:
Thanks, done in 15948491b6